src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -294,6 +295,22 @@ class DeployTask extends AbstractServerTask {
         if (container) {
             config.setProjectRoot(task.getProject().getProjectDir().getAbsolutePath());
```

Comment:
Although this line wasn't changed from this PR, use getCanonicalPath here to be consistent with the below, similar to https://github.com/OpenLiberty/ci.maven/blob/master/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java#L118

Reply:
I agree.